backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -220,9 +220,30 @@ public String getResultID() {
 		return testEvent.getInternalId().toString();
 	}
 
+	// 27Jan2021 Added
+	@JsonProperty("Corrected_result_ID")
+	public String getCorrectedResultId() {
+		if (testEvent.getCorrectionStatus() != TestCorrectionStatus.ORIGINAL) {
+			return testEvent.getPriorCorrectedTestEventId().toString();
+		}
+		return "";
+	}
+
+	// 27Jan2021 Updated to handle deleted tests
 	@JsonProperty("Test_result_status")
 	public String getTestResultStatus() {
-		return testResultStatusFinal;
+		// F Final results
+		// X No results available; Order canceled
+		// C Corrected, final (not yet supported
+		switch(testEvent.getCorrectionStatus()) {
+			case REMOVED:
+				return "X";
+			case CORRECTED:
+				return "C";
+			case ORIGINAL:
+			default:
```

Comment:
Note that the default case almost never does anything useful, since switching on a `null` is not allowed.